### PR TITLE
Rename PowerShell cmdlets module to EntityFrameworkCore

### LINKED ISSUE #5145: Migrations PowerShell commands can't work for both EF6 and EF Core projects that are part of the same solution
The PowerShell versions of the migrations commands are global to the solution and have the same names in EF6 and EF Core but their implementations are version specific. Only one version of them can be active per solution. This can be very inconvenient while incrementally migrating existing projects to EF Core if at any point projects targeting different versions of EF are part of the same solution.

Possible workarounds customers could use are:
- Uninstall and reinstall the EF packages as needed
- Split the solution (more permanent)

In a quick brainstorming we came up with a few ideas (we haven't validated feasibility):
- Release commands that are compatible with both EF6 and EF Core
- Provide an easy way to unload or swap the active PS module

### PR DESCRIPTION
**Pull request by natemcmaster**
Fix #5145

Related: documenation update https://github.com/aspnet/EntityFramework.Docs/pull/170

~~Btw, we still have "Apply-Migrations" and "Enable-Migrations" in this module. Can we remove these now? Or do we still want them there to throw "obsolete" errors?~~ nevermind. Just realized Brice opened an issue a few hours before me to track this exact question.

cc @rowanmiller @bricelam @divega

### CODE REVIEW

File: src/Microsoft.EntityFrameworkCore.Tools/tools/EntityFrameworkCore.psd1 [hunk 1]
```diff
@@ -58,14 +58,15 @@
...
-        'Add-Migration',
+        'Enable-Migrations',
         'Apply-Migration',
-        'Update-Database',
-        'Script-Migration',
+        
+        'Add-Migration',
         'Remove-Migration',
-        'Enable-Migrations',
         'Scaffold-DbContext'
```
Comment (lajones) [inline src/Microsoft.EntityFrameworkCore.Tools/tools/EntityFrameworkCore.psd1]: Does the Scaffold-DbContext line now need a comma?

Comment (natemcmaster) [inline src/Microsoft.EntityFrameworkCore.Tools/tools/EntityFrameworkCore.psd1]: Oops. Yes. Will fix.

**Comment by bricelam** [comment]
:shipit:

File: src/Microsoft.EntityFrameworkCore.Tools/tools/about_EntityFramework.help.txt [hunk 2]
```diff
@@ -7,14 +7,14 @@
...
+    about_EntityFrameworkCore
 
 SHORT DESCRIPTION
-    Provides information about Entity Framework commands.
+    Provides information about Entity Framework Core commands.
 
 LONG DESCRIPTION
-    This topic describes the Entity Framework commands. Entity Framework is Microsoft's recommended data access technology for new
-    applications.
+    This topic describes the Entity Framework Core commands. Entity Framework is Microsoft's recommended data access technology
```
Comment (divega) [inline src/Microsoft.EntityFrameworkCore.Tools/tools/about_EntityFramework.help.txt]: I know this is merged already, sorry I still need to comment... I was thinking how we can fix it, but can we get rid of the second sentence altogether? Maybe a link to our documentation page would be more useful 

cc @rowanmiller

Comment (rowanmiller) [inline src/Microsoft.EntityFrameworkCore.Tools/tools/about_EntityFramework.help.txt]: Agreed...

> This topic describes the Entity Framework Core commands. See https://docs.efproject.net for information on Entity Framework Core.

**Pull request merged**